### PR TITLE
refactor: minor adjustments after protocol 6.1 support

### PR DESCRIPTION
**Pull request by mathpaquette**

### CODE REVIEW

**Comment by mathpaquette** [comment]
@NichUK okay just got why you opted for the previous approach...

File: src/IQFeed.CSharpApiClient.Tests/Lookup/Historical/Messages/TickMessageTests.cs [hunk 1]
```diff
@@ -136,20 +136,5 @@ public void Should_TryParse_Return_True(string cultureName)
             Assert.IsTrue(parsed);
             Assert.AreEqual(tickMessage, _expectedMessage);
         }
-
-        [Test, TestCaseSource(typeof(CultureNameTestCase), nameof(CultureNameTestCase.CultureNames))]
-        public void Should_TryParse_Protocol60_Return_True(string cultureName)
```
Comment (NichUK) [inline src/IQFeed.CSharpApiClient.Tests/Lookup/Historical/Messages/TickMessageTests.cs:141]: I don't understand why you're getting rid of this test. It tests something different to the others.

File: src/IQFeed.CSharpApiClient/Lookup/Historical/HistoricalRequestFormatter.cs [hunk 2]
```diff
@@ -58,13 +58,12 @@ public string ReqHistoryTickDatapoints(string symbol, int maxDatapoints, DataDir
...
-            days = days > Int16.MaxValue ? Int16.MaxValue : days;
             var beginFilterTimeFormat = beginFilterTime?.ToString(HistoricalDataTimeFormat) ?? string.Empty;
             var endFilterTimeFormat = endFilterTime?.ToString(HistoricalDataTimeFormat) ?? string.Empty;
             var dataDirectionFormat = dataDirection.HasValue ? ((int)dataDirection).ToString() : null;
             var intervalTypeFormat = intervalType?.ToString().ToLower();
             var labelAtBeginningFormat = labelAtBeginning.HasValue ? ((int)labelAtBeginning).ToString() : null;
-            var request = $"HID,{symbol.ToUpper()},{interval},{days},{maxDatapoints},{beginFilterTimeFormat},{endFilterTimeFormat},{dataDirectionFormat},{requestId},{datapointsPerSend},{intervalTypeFormat},{labelAtBeginningFormat}{IQFeedDefault.ProtocolTerminatingCharacters}";
+            var request = $"HID,{symbol.ToUpper()},{interval},{days.ToMaximumDaysWeeksMonths()},{maxDatapoints},{beginFilterTimeFormat},{endFilterTimeFormat},{dataDirectionFormat},{requestId},{datapointsPerSend},{intervalTypeFormat},{labelAtBeginningFormat}{IQFeedDefault.ProtocolTerminatingCharacters}";
             return request;
```
Comment (NichUK) [inline src/IQFeed.CSharpApiClient/Lookup/Historical/HistoricalRequestFormatter.cs:67]: personally I thought the other was more readable, but yours is DRYer. :)

File: src/IQFeed.CSharpApiClient/Lookup/Historical/Messages/TickMessage.cs [hunk 3]
```diff
@@ -97,43 +119,21 @@ public static bool TryParse(string message, out TickMessage tickMessage)
...
-                             long.TryParse(values[6], NumberStyles.Any, CultureInfo.InvariantCulture, out tickId) &&
-                             char.TryParse(values[7], out basisForLast) &&
-                             int.TryParse(values[8], NumberStyles.Any, CultureInfo.InvariantCulture, out tradeMarketCenter) &&
-                             int.TryParse(values[10], NumberStyles.Any, CultureInfo.InvariantCulture, out tradeAggressor) &&
-                             int.TryParse(values[11], NumberStyles.Any, CultureInfo.InvariantCulture, out dayCode);
-
-                if (!parsed)
-                    return false;
-            }
+            var parsed = values.Length >= 10 &&
```
Comment (NichUK) [inline src/IQFeed.CSharpApiClient/Lookup/Historical/Messages/TickMessage.cs:122]: you changed the previous constant to TickCsvLengthProtocol60 so this should be likewise

**Review by NichUK — COMMENTED** [review]
Most of that looks very reasonable tidying. 
Just concerned about removing the test, if we're still supporting data in the old format. We still need to be able to parse both version, right?

**Comment by NichUK** [comment]
It seemed reasonably straight-forward and performant as I had it, but if it can be improved then I'm totally up for it. :)

File: src/IQFeed.CSharpApiClient.Tests/Lookup/Historical/Messages/TickMessageTests.cs (hunk 1 above)
Comment (mathpaquette) [reply src/IQFeed.CSharpApiClient.Tests/Lookup/Historical/Messages/TickMessageTests.cs:141]: well, this one should technically only be used when requesting new data. Older data should necessary come from disk.

Comment (NichUK) [reply src/IQFeed.CSharpApiClient.Tests/Lookup/Historical/Messages/TickMessageTests.cs:141]: So we should move the test to a different set? HistoricalFileFacadeTests? It's still worth testing somewhere though, isn't it?

Comment (mathpaquette) [reply src/IQFeed.CSharpApiClient.Tests/Lookup/Historical/Messages/TickMessageTests.cs:141]: well, in fact, ill revert back most my changes for TickMessages, yours seems totally okay.

**Pull request closed without merging**